tensorflow_model_optimization/g3doc/guide/clustering/clustering_with_keras.ipynb

```diff
@@ -0,0 +1,1164 @@
+{
```

Comment:
<p>References aren't a part of this document.</p><p> </p><p>Anything on "do you want to use clustering" and "the non-demonstrated benefits of clustering" should live in the overview page. This page is to help the user understand how to use clustering in one case, while demonstrating certain benefits. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/model-optimization/pull/330/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='60'/>